package.json

```diff
@@ -17,9 +17,5 @@
 , "devDependencies": {
     "mocha":"*"
 }
-, "licenses" :
-  [ { "type" : "MIT"
-    , "url" : "http://github.com/sivy/node-stats/raw/master/LICENSE"
-    }
-  ]
+, "license" : "MIT"
```

Comment:
Unrelated, and if you want this in a separate PR (or just don't like it for some reason) I can remove it.  It's just something I noticed when doing an npm install that there was a warning.  npm has switched to this license format as described in https://docs.npmjs.com/files/package.json